iree/compiler/Translation/HALExecutable.h

```diff
@@ -0,0 +1,24 @@
+// Copyright 2021 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#ifndef IREE_COMPILER_TRANSLATION_HALEXECUTABLE_H_
+#define IREE_COMPILER_TRANSLATION_HALEXECUTABLE_H_
+
+#include "iree/compiler/Dialect/HAL/Target/TargetRegistry.h"
+#include "llvm/Support/raw_ostream.h"
+#include "mlir/IR/BuiltinOps.h"
+#include "mlir/Pass/PassManager.h"
+#include "mlir/Support/LogicalResult.h"
```

Comment:
Is there some technical or style reason for these includes in the header file? With how empty this header is (just a single forward declaration), only the .cpp really needs includes. I've been pruning includes from files like this that I come across, suspecting bit rot + copy paste :P